src/convertToObject.js

```diff
@@ -6,7 +6,25 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const FORMAT_SOURCE = sourceString.split('\n').join('');
+  const STYLE_ARRAY = FORMAT_SOURCE.split(';');
+
+  const STYLE_OBJECT = {};
+
+  const PAIRS_OF_STYLE = STYLE_ARRAY.map((item) => item.split(':'));
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. Here, you could directly use a `reduce` method to transform the array into an object, which would be more efficient and concise.